spec/services/calculators/multiple_employments_calculator_spec.rb

```diff
@@ -6,6 +6,8 @@ module Calculators
     let(:result) { described_class.call(assessment.submission_date) }
 
     it "sets all items to zero apart from allowance" do
+      expect(result.employment).to be_entitles_employment_allowance
+      expect(result.employment).not_to be_entitles_childcare_allowance
```

Comment:
This doesn't read very nicely - could we not just check the attributes?

Reply:
@starswan Its a rubocop suggestion. Initially i tried with `expect(result.employment.entitles_employment_allowance?).to be_truthy`